Brewfile.lock.json

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
This is based on the `brew bundle` execution - if anyone else runs it subsequently, they will get these versions. Happy to peg versions in the `Brewfile` if need be.